common/metrics/consts.go

```diff
@@ -0,0 +1,35 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package metrics
+
+type MetricUnit string
+
+// MetricUnit supported values
+// Values are pulled from https://pkg.go.dev/golang.org/x/exp/event#Unit
+const (
+	Dimensionless = "1"
+	Milliseconds  = "ms"
+	Bytes         = "By"
```

Comment:
should the consts be typed?
```suggestion
	Dimensionless MetricUnit = "1"
	Milliseconds MetricUnit  = "ms"
	Bytes MetricUnit        = "By"
```

Reply:
I can change in a separate PR. I notice this, but it will require quite some changes around the code base.

Reply:
ok. not a big deal, I just thought it might be easy